src/components/CCPDashboard/DeletePatientDialog.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {
+  DialogContent,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogTitle,
+} from '@material-ui/core';
+import { Colours } from '../../styles/Constants';
+
+const DeletePatientDialog = ({
```

Comment:
we now have a common `ConfirmModal` which may be useful to use
https://github.com/uwblueprint/paramedics-react/blob/master/src/components/common/ConfirmModal.tsx